tests/e2e/e2e_test.go

```diff
@@ -44,6 +44,32 @@ func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 	chainB.SendIBC(chainA, chainA.NodeConfigs[0].PublicAddress, initialization.StakeToken)
 }
 
+func (s *IntegrationTestSuite) TestRateLimitingTestsSetupCorrectly() {
+	// Checking the rate limiting tests are setup correctly
+	//f1, err := ioutil.ReadFile("../../x/ibc-rate-limit/testdata/rate-limiter.wasm")
+	//s.NoError(err)
```

Comment:
Suggest trying `s.Require().NoError(err)`

Without it, the test does not fail and continues.

It seems that there might be a path/permission issue at this step with:
```
open ../../x/ibc-rate-limit/testdata/rate-limiter.wasm: no such file or directory
```

Reply:
yes, sorry! that test was just me experimenting with making sure the file we're using for e2e tests (the one copied to the docket containers) is the same as the one in the integration tests. The path should be `/osmosis/rate-limiter.wasm`.

Reply:
Fixed. Moved these tests to the integration tests (since they don't depend on the e2e containerization). 

I also want to add a test to check the rust code compiles to the wasm file we're using for testing, but I'm not sure if the go tests is the right place to do this. Seems like something that should be its own task in the workflow (especially if more contracts are going to start being part of the source)

Reply:
Awesome! I think a separate task in the workflow makes sense